projects/kubernetes-sigs/image-builder/packer/ami/packer.json

```diff
@@ -5,5 +5,6 @@
   "snapshot_groups": "",
   "custom_role": "true",
   "custom_role_names": "{{env `BUILDER_ROOT`}}/ansible/roles/load_additional_files",
-  "ansible_extra_vars": "@{{env `BUILDER_ROOT`}}/packer/ami/ansible_extra_vars.yaml"
+  "ansible_extra_vars": "@{{env `BUILDER_ROOT`}}/packer/ami/ansible_extra_vars.yaml",
+  "volume_size": "30"
```

Comment:
Correct me if Im wrong, doesn't upstream set the default size to be 25G? Maybe not for ami packer config.

In either case, do we want to stick with 25G across all providers? I'm down for ami to be 30 if that's something we really want.

Reply:
The default size for this AMI was 8GB. Tbh 30GB was kind of arbitrary. I'm ok with 25 is that is more common.

Reply:
yea lets do 25, just to be consistent across all providers.

Reply:
Done